concordion/src/test/resources/nl/knaw/huc/textrepo/task/TestImportDocument.md

```diff
@@ -0,0 +1,185 @@
+# Test `/task/import/documents/{externalId}/{typeName}`
+
+File contents can be directly uploaded to a document, referenced by its `externalId`. This yields a (possibly new)
+version of a `typeName` typed file for that document.
+
+If the document with `externalId` does not yet exist, you can have the repository create it during import by passing
+`allowNewDocument=true`.
+
+Otherwise, using `allowNewDocument=false`, the import will verify that you are uploading to an already existing
+document, or deny the request. Note that this is also the default if you leave `allowNewDocument` out.
+
+If you offer the same contents of a version already registered for this document during another import, by
```

Comment:
Duidelijke uitleg
Misschien kan een aantal keer 'during/by another import' nog weg? Zonder leest het wat makkelijker, en het voegt verder volgens mij niets toe aan de uitleg?

Reply:
Deprolixified in a6c8a11.

Reply:
Deprolixification confirmed